Programowanie Obiektowe/1.2Klasy/1.2Klasy/Point3D.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _1._2Klasy
+{
+    internal class Point3D
+    {
+
+        public int X { get; set; }
```

Comment:
Nie wiem jaka jest treść zadania ale możesz zablokować możliwość ustawiania właściwości z zewnątrz. Jest na to kilka sposobów. Najlepiej byłoby uzyć [records](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/records) albo
```suggestion
        public int X { get; }
```
albo
```suggestion
        public int X { get; private set; }
```
Każde się różni. Dobrze by było żebyś wiedział czym.

Reply:
[a235a4c](https://github.com/broniuu/Praktyki2022/pull/2/commits/a235a4ce7a09e888c1c932cfe88452242b5eeb32)